src/libraries/libSCL_RIP7212.sol

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
nit: I'd add a comment in the code itself about why we've copied it to this repo and the changes made